caluma-demo/app/router.js

```diff
@@ -0,0 +1,18 @@
+import EmberRouter from '@ember/routing/router';
+import config from './config/environment';
+
+const Router = EmberRouter.extend({
+  location: config.locationType,
+  rootURL: config.rootURL
+});
+
+Router.map(function() {
+  this.mount("ember-caluma", {
+    as: "form-builder",
+    path: "/form-builder"
+  });
+  this.route('form-builder');
```

Comment:
This is a duplicated route

Reply:
I have noticed this myself and have already deleted the route.